recipe/conda_build_config.yaml

```diff
@@ -5,23 +5,9 @@ MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
 folly_build_ext:
   - None
   - "jemalloc"  # [not win]
-fmt:
-  - "10"
-  - "9"
-  - "9"
+libboost_devel:
+  - "*"
+  - "1.84.0"
 libevent:
-  - "2.1.12"
+  - "*"
```

Comment:
Please pin to an explicit version here. Otherwise, this will build for what's latest. We normally don't want that.